### PR TITLE
Correct various typos

### PR DESCRIPTION
**Pull request by goggle**
### Purpose
Corrects some typos that I've found in the repository.

### CODE REVIEW

**Comment by calmh** [comment]
Half of this is in autogenerated files. If you avoid those, the rest makes sense.

**Review by acolomb — APPROVED** [review]
Looks good in code, but I haven't tested it or checked whether all instances of the variable renames were caught.

File: man/syncthing-rest-api.7 [hunk 1]
```diff
@@ -1994,7 +1994,7 @@ These endpoints require the \fBgui.debugging\fP configuration option to
 be enabled and yield an access denied error code otherwise.
 .SS GET /rest/debug/peerCompletion
 .sp
-Summarizes the completion precentage for each remote device.  Returns an object
```
Comment (calmh) [inline man/syncthing-rest-api.7:1997]: This file is autogenerated from the docs repo

**Review by calmh — CHANGES_REQUESTED** [review]
(Remove changes to autogenerated files)

**Review by calmh — APPROVED** [review]

**Pull request merged**